docs/articles/documentation/extending-testcafe/reporter-plugin/reporter-methods.md

```diff
@@ -126,6 +130,7 @@ The `testRunInfo` object provides detailed information about the test run. The o
 Property            | Type             | Description
 ------------------- | ---------------- | --------------------------------------------------------
 `errs`              | Array of Strings | An array of errors that occurred during the test run.
```

Comment:
I forgot to correct this error from our current docs. `errs` is not an `Array of Strings`, but an `Array of Objects`. TestCafe errors are special objects and you should use the `formatError` helper to transform them to message strings.

Reply:
Has it always been so, or was it introduced as a breaking change at some stage?
Do we need to detail this object's structure or the `formatError` helper will be enough?

Reply:
Okay, we decided to document only `formatError`

Reply:
I guess it was implemented that way, We will need a lot of time to properly describe its internal structure because there is a lot of error classes with different sets of properties.